src/opm/parser/eclipse/EclipseState/Schedule/Well/WellTestConfig.cpp

```diff
@@ -121,6 +130,16 @@ std::string WellTestConfig::reasonToString(const Reason reason) {
 
 
 
+WellTestConfig::WTESTWell*  WellTestConfig::getWell(const std::string& well_name, const Reason reason) {
+    const auto well_iter = std::find_if(wells.begin(), wells.end(), [&well_name, &reason](const WTESTWell& well) {
+        return (reason == well.shut_reason && well.name == well_name);
+    });
+
+    return (well_iter == wells.end() ? nullptr : &(*well_iter) );
```

Comment:
Maybe use `std::addressof( )` instead of `&` to take the address.

Reply:
I believe the std::addressof() function is designed to handle weird cases when writing a template and you want a `T*` but cannot be sure unary `&` will give you one due to possible overloading. It is not necessary here. However I get the feeling you think it gives more readable code? There is also the (remote and weird) possibility that a class is overloaded on `&` and the result of that (or a side effect) is what we actually want (otherwise why overload it), in which case std::addressof() will overrule that to produce a genuine `T*`.

Reply:
>  However I get the feeling you think it gives more readable code? 

Just looks nicer - no big deal.